src/modules/75-cf/components/CreateEnvironmentDialog/EnvironmentDialog.tsx

```diff
@@ -150,100 +148,46 @@ const EnvironmentDialog: React.FC<EnvironmentDialogProps> = ({ disabled, onCreat
           {formikProps => {
             return (
               <FormikForm>
-                <Container
-                  padding="xxxlarge"
-                  style={{
-                    height: '100%',
-                    display: 'flex',
-                    flexDirection: 'column',
-                    paddingBottom: 'var(--spacing-medium)'
-                  }}
-                >
-                  <Text font={{ size: 'medium', weight: 'bold' }} color={Color.BLACK}>
-                    {getEnvString('create.title')}
-                  </Text>
-                  <Text margin={{ top: 'large', bottom: 'large' }}>{getEnvString('create.description')}</Text>
-                  <Layout.Vertical
-                    spacing="small"
-                    style={{ minHeight: 415, overflow: 'auto', padding: 'var(--spacing-xsmall)' }}
-                  >
-                    <FormInput.InputWithIdentifier
-                      inputName="name"
-                      idName="identifier"
-                      isIdentifierEditable
-                      inputLabel={getEnvString('create.nameLabel')}
-                      inputGroupProps={{ inputGroup: { autoFocus: true } }}
-                    />
-                    <Layout.Vertical>
-                      <Container className={css.collapse}>
-                        <Collapse
-                          {...collapseProps}
-                          heading={getString('description')}
-                          collapseHeaderClassName={css.collapseHeaderFix}
+                <Text font={{ size: 'small' }}>{getEnvString('create.description')}</Text>
```

Comment:
Can we swap this out for a variation?
```suggestion
                <Text font={{ variation: FontVariation.SMALL }}>{getEnvString('create.description')}</Text>
```